src/platform/intel/cavs/platform.c

```diff
@@ -488,6 +488,10 @@ int platform_init(struct sof *sof)
 	shim_write16(SHIM_PWRCTL, SHIM_PWRCTL_TCPDSP0PG);
 #endif
 
+#if defined(CONFIG_SUECREEK)
+	dw_uart_init();
```

Comment:
This would be a generic uart = uart_init(any params) and we would use a CONFIG_CAVS_UART (which selects DW_UART in Kconfig) instead of CONFIG_USE_CREEK